napari/layers/labels/labels.py

```diff
@@ -507,7 +512,7 @@ def mode(self, mode: Union[str, Mode]):
             self.help = 'hold <space> to pan/zoom, click to pick a label'
             self.mouse_drag_callbacks.append(pick)
         elif mode == Mode.PAINT:
-            self.cursor_size = self.brush_size / self.scale_factor
+            self.brush_size = self.brush_size
```

Comment:
This is super confusing to read. At first I thought that we could just delete this line, but seeing the setter above I realise we are using the setter for its side effects.

That's a bad smell, but we can hold our noses for a bit — as long as we add a comment above this line that this is what we are doing!

Reply:
ok, comment added, I think I might be able to fix this in #1763 before that merges